internal/slicer/slicer_test.go

```diff
@@ -755,8 +779,95 @@ var slicerTests = []slicerTest{{
 	},
 	error: `slice test-package_myslice: content is not a file: /x/y`,
 }, {
-	summary: "Non-default archive",
+	summary: "Multiple archives with priority",
+	slices:  []setup.SliceKey{{"test-package", "myslice"}, {"other-package", "myslice"}},
+	archives: map[string]*testArchive{
+		"foo": {
+			pkgs: map[string][]byte{
+				"test-package": testutil.MustMakeDeb([]testutil.TarEntry{
+					testutil.Reg(0644, "./file", "from foo"),
+				}),
+			},
+		},
+		"bar": {
+			pkgs: map[string][]byte{
+				"test-package": testutil.MustMakeDeb([]testutil.TarEntry{
+					testutil.Reg(0644, "./file", "from bar"),
+				}),
+				"other-package": testutil.MustMakeDeb([]testutil.TarEntry{
+					testutil.Reg(0644, "./other-file", "from bar"),
+				}),
+			},
+		},
+	},
+	release: map[string]string{
+		"chisel.yaml": `
+			format: chisel-v1
+			archives:
+				foo:
+					version: 22.04
+					components: [main, universe]
+					priority: 20
+					v1-public-keys: [test-key]
+				bar:
+					version: 22.04
+					components: [main]
+					default: true
+					priority: 10
+					v1-public-keys: [test-key]
+			v1-public-keys:
+				test-key:
+					id: ` + testKey.ID + `
+					armor: |` + "\n" + testutil.PrefixEachLine(testKey.PubKeyArmor, "\t\t\t\t\t\t") + `
+		`,
+		"slices/mydir/test-package.yaml": `
+			package: test-package
+			slices:
+				myslice:
+					contents:
+						/file:
+		`,
+		"slices/mydir/other-package.yaml": `
+			package: other-package
+			slices:
+				myslice:
+					contents:
+						/other-file:
+		`,
+	},
+	filesystem: map[string]string{
+		// /file comes from the test-package in archive "foo" as archive "foo"
+		// has a higher priority, althought archive "bar" is the default
+		// archive. This demonstrates that in light of the new "priority" value,
+		// the notion of "default" archives is now obsolete.
+		"/file": "file 0644 7a3e00f5",
+		// /other-file comes from the other-package in archive "bar" as archive
+		// "bar" is the only one which has the package.
```

Comment:
[Nit]: Can you change the comments a little to make it more uniform with the existing style in `slicer_test.go`?